discovery_engine_core/web-api/Cargo.toml

```
@@ -3,36 +3,38 @@ name = "web-api"
 version = { workspace = true }
 edition = { workspace = true }
 rust-version = { workspace = true }
+description = "Wep Api server for the document personalization service."
 license = "AGPL-3.0-only"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bincode = { workspace = true }
-bytes = { version = "1.2.1", features = ["serde"] }
-chrono = { workspace = true, features = ["serde"] }
-derive_more = { workspace = true }
+actix-web = { version = "4.2.1", default-features = false, features = ["compress-gzip", "compress-zstd"] }
+anyhow = { workspace = true }
+clap = { workspace = true }
+derive_more = { workspace = true, features = ["display", "deref", "as_ref", "into"] }
 displaydoc = { workspace = true }
 dotenvy = "0.15.5"
-envconfig = "0.10.0"
-futures = { workspace = true }
+figment = { workspace = true, features = ["env", "toml"] }
+futures-util = { workspace = true }
 itertools = { workspace = true }
-ndarray = { workspace = true }
+mime = "0.3.16"
 once_cell = { workspace = true }
+pin-project = { workspace = true }
 regex = { workspace = true }
 reqwest = { workspace = true }
+secrecy = { version = "0.8.0", features = ["serde"] }
 serde = { workspace = true }
 serde_json = { workspace = true }
 sqlx = { workspace = true, features = ["postgres"] }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
 tracing = { workspace = true }
-tracing-subscriber = { workspace = true, features = ["env-filter"] }
+tracing-subscriber = { workspace = true, features = ["env-filter", "json"] }
```

🔧 the second feature is already included in the workspace `Cargo.toml`
```suggestion
tracing-subscriber = { workspace = true, features = ["env-filter"] }
```

---

I will add this to 

- #691